Tajaran Fix: Custom Species & Missing Markings Slots

Cprn-Sprkl opened the pull request:
# Description

Adds the one line needed to allow for custom species while selecting Tajaran.

---

# Changelog

:cl: sprkl
- fix: Tajaran can have their species name customized.

Cprn-Sprkl committed:
Tajaran Fix: Custom Species

Adds the one line needed to allow for custom species while selecting Tajaran.

M3739 commented:
Actually, there is something you can do for me real quick.

I just tried out the Tajarans today, and they are lacking their underwear and undershirt marking slots as well. Think you can add that fix onto this PR?

Cprn-Sprkl commented:
Not sure why it isn't working. Haven't worked much with the marking and layer systems, but on a quick check I'm not seeing anything special humans have that allow them to have underwear where Tajaran do not. They both inherit from the same parent which looks like it has the underwear stuff. I need to do some work on markings in general for Tajaran so it might be best to just take this PR as is, and I do the face markings, tail layering, and underwear stuff in a separate PR.

M3739 commented:
> Not sure why it isn't working. Haven't worked much with the marking and layer systems, but on a quick check I'm not seeing anything special humans have that allow them to have underwear where Tajaran do not. They both inherit from the same parent which looks like it has the underwear stuff. I need to do some work on markings in general for Tajaran so it might be best to just take this PR as is, and I do the face markings, tail layering, and underwear stuff in a separate PR.

Sounds good. It may indeed be related to the marking layering issue.

Cprn-Sprkl committed:
Upgrade Tajaran Markings

Allows Tajaran to correctly use face, underwear, undershirt, and wing markings. Also adds the TailBehind and TailOversuit marking layers for use with the tail split system.

Cprn-Sprkl commented:
Had some time to study the markings stuff and found what was missing for Tajaran. As far as I can tell all markings now work, including tails that were set up to work with the split system. Grunt work will have to be done to upgrade all tails to work with this system, but that's not Tajaran specific and would edit a lot of RSI files, so a task for a different PR.

M3739 commented:
Gotcha. For now, this looks good.
Just need another to approve.

Mnemotechnician commented:
> Had some time to study the markings stuff and found what was missing for Tajaran. As far as I can tell all markings now work, including tails that were set up to work with the split system. Grunt work will have to be done to upgrade all tails to work with this system, but that's not Tajaran specific and would edit a lot of RSI files, so a task for a different PR.

There are actually dozens of tails that need changing across all species. In my PR I only touched the 3 standard vulpkanin tails, and that took me a fair bit of grunt work. I would recommend to write a script to do the split automatically instead of doing it manually.
